### DISCUSSION TITLE
New release (?)

### DISCUSSION THREAD

**Discussion by isaranto** [catboost/catboost#2592] [General]
Hello! I just wanted to ask if there are any news regarding the latest release. We are interested in using the latest code that has  with the [cgroupsv2 support](https://github.com/catboost/catboost/issues/2518) and was wondering if we should wait.

Thanks!

**Comment by andrey-khropov** [comment]
I expect it to be released this week. Sorry for the wait but there have been some critical issues to be solved.

**Comment by mkleinbort-ic** [comment]
Just chiming in - any luck with a release this week? In the middle of moving to 3.12 but blocked on this. I could wait a day or two if it's about to be released, else I'll plan something else this week.

**Reply by Iqigai** [reply]
I wanted to move to 3.12 as well but got an issue with the installation of catboost. Is it related?

**Reply by andrey-khropov** [reply]
Yes, I'm working on it right now.

**Comment by andrey-khropov** [comment]
> I wanted to move to 3.12 as well but got an issue with the installation of catboost. Is it related?

If you try to install version 1.2.2 it most likely won't work, installing from the recent revision from the master branch from GitHub should work (although not recommended for production).

**Comment by einarwar** [comment]
Sorry for bothering, I am sure you have lots of work, but do you have any updates on the release of 1.2.3? Catboost is currently our only blocker to upgrade to python 3.12

Thanks

**Reply by andrey-khropov** [reply]
I expect it to be released today, the last issues have been resolved.

**Comment by andrey-khropov** [comment]
[CatBoost 1.2.3 has been released](https://github.com/catboost/catboost/releases/v1.2.3)
